index.js

```diff
@@ -102,3 +100,50 @@ module.exports.isMACValid = function(mac) {
 
     return !(mac.length != 2 * MAC_BYTES || mac.match(/[^a-fA-F0-9]/));
 }
+
+module.exports.relay = function(mac, params) {
+
+        getIP(params).then(function(ip) {
+            
+            var magicPacket = createMagicPacket(mac);
+            listensocket = dgram.createSocket('udp4');
+            
+            listensocket.on('error', (err) => {
```

Comment:
if you are using es6, please change node version in package.json

Reply:
I got this code from an example that I found.

I'm quite new to JS, and not too familiar with what has been updated in ES6 and how that affects the node version.  Could you tell me what about this is requires ES6, and the implications to node version?

Reply:
I see, you mean the arrow function here.  Do you have any objection to updating the node version, then?

Reply:
I think, we can easily upgrade to latest node.js version :)